### ISSUE TITLE
simultaneous vs. 2-key sequence

### ISSUE THREAD

**Issue by Jimmy34742** [tekezo/Karabiner-Elements#1268]
My launcher uses 20 keys on the left side of the keyboard to enter 20 different modes, and then I hit a key on the right side to launch the desired item. In other words, I launch all my stuff with 2-key codes, always pressing the key on the left first. I use 2-key codes exclusively so that I can have identical launch sequences on my Windows computer using AutoHotKey, which doesn't allow sequences of greater than two keys.)

With the new simultaneous keypress capability in the latest beta, I could convert my launcher to all 2-key simultaneous presses. In terms of physical typing, there doesn't seem much difference between doing 2-key left/right sequences versus 2-key simultaneous combos. But can anyone comment on the costs and benefits of going all-simultaneous vs. going all-modes? I expect I'll try it for myself and see, but before I do all that editing, I wonder if anyone has an opinion as to efficiency, practical usability, side effects, etc.

**Comment by prohtex** [comment]
@Jimmy34742 I'm also trying to do this. Do you have any examples? There are plenty for simultaneous, but I want to use a sequence as the trigger.

**Comment by Jimmy34742** [comment]
For my two-key launcher, I've discovered there's no question that using "simultaneous" exclusively is definitely the way to go.  Setting variables to create modes requires making sure to fully separate the mode-key presses. When you have 20 modes, your overall typing has to become fully separated, as the keyboard becomes highly intolerant to any overlapping. If you overlap at all when typing text, keystrokes are missed. I modified my typing technique and was getting better at it, but the zero tolerance for overlapping was annoying. I re-did my two-key interface as all-simultaneous, and everything is much more comfortable. I won't go back, though my typing technique has been improved by the experience.

For using sequences as triggers, if I understand you correctly, wouldn't using "simultaneous" with the various ordering options be what you're looking for? If that's only available in the beta, then get on the beta update channel.
